tokens/56.json

```diff
@@ -43,27 +37,21 @@
       "decimals": 18
     },
     {
-      "token_address": "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c",
-      "name": "BTCB Token",
-      "symbol": "BTCB",
-      "decimals": 18
-    },
-    {
-      "token_address": "0x53E63a31fD1077f949204b94F431bCaB98F72BCE",
-      "name": "SolvBTC Ethena",
-      "symbol": "SolvBTC.ENA",
+      "token_address": "0x1af3f329e8be154074d8769d1ffa4ee058b1dbc3",
+      "name": "Dai Token",
+      "symbol": "DAI",
       "decimals": 18
     },
     {
-      "token_address": "0x4aae823a6a0b376De6A78e74eCC5b079d38cBCf7",
-      "name": "Solv BTC",
-      "symbol": "SolvBTC",
+      "token_address": "0x5f0da599bb2cccfcf6fdfd7d81743b6020864350",
+      "name": "Maker",
+      "symbol": "MKR",
       "decimals": 18
     },
     {
-      "token_address": "0x1346b618dC92810EC74163e4c27004c921D446a5",
-      "name": "SolvBTC Babylon",
-      "symbol": "SolvBTC.BBN",
+      "token_address": "0x52ce071bd9b1c4b00a0b92d298c512478cad67e8",
+      "name": "Compound Coin",
+      "symbol": "COMP",
       "decimals": 18
     }
   ],
```

Comment:
what protocols support this on BSC? Pretty certain I didn't see these tokens via integrating and using dapps.

Also guessing these tokens are removed because of Moralis?

Reply:
yes, correct, i vailadate with pricefeeder